pkg/utils/common/rbac.go

```diff
@@ -120,3 +120,13 @@ func (s *SuiteController) CreateRoleBinding(roleBindingName, namespace, subjectK
 	}
 	return createdRoleBinding, nil
 }
+
+// DeleteRoleBinding deletes the role binding with the provided name and namespace
+func (s *SuiteController) DeleteRoleBinding(roleBindingName, namespace string) error {
+	// if rolebinding is not found, return nil
+	_, err := s.GetRoleBinding(roleBindingName, namespace)
+	if err != nil {
+		return nil
```

Comment:
To ensure that the rolebinding was not found, we should check `err.IsNotFound`, imho. WDYT?

Reply:
thanks for your suggestion. fixed